lib/config/migrations/custom/package-rules-migration.ts

```diff
@@ -25,6 +25,29 @@ function renameKeys(packageRule: PackageRule): PackageRule {
   return newPackageRule;
 }
 
+function removeMatchLanguage(packageRule: PackageRule): PackageRule[] {
+  const newPackageRules: PackageRule[] = [];
+  const matchLanguages = packageRule.matchLanguages;
+  delete packageRule.matchLanguages;
```

Comment:
```suggestion
  const matchLanguages = packageRule.matchLanguages ?? packageRule.languages;
  delete packageRule.matchLanguages;
  delete packageRule.languages;
```

this should fix my comment above but needs a new test case.